document.txt

```diff
@@ -0,0 +1,8 @@
+This is an important document 
+-------------
+
+Here's an important list:
+1) Important item
+2) Very important item
+3) Less important item, but still important
+4) Extremely important item

```

Comment:
Actually, it's ok

---
In reply to: [54176729](https://github.com/kivancmuslu/CFTest/pull/2#discussion_r54176729) [](ancestors = 54176729)